core/modules/file/file.module

```diff
@@ -1316,6 +1316,9 @@ function file_managed_file_process($element, &$form_state, $form) {
   );
 
   // Add a button that will be used to select existing files if enabled.
+  // Attaching the assets from filter module makes sure, the browser works
+  // even if no field on that page has a filter set.
```

Comment:
I feel like this comment should be moved down to just above the `#attached` line (where it's referring to). Also, remove the comma please.